receiver/dockerstatsreceiver/documentation.md

```diff
@@ -306,6 +306,14 @@ This metric is only reported if the container has limits set with -cpus, -cpuset
 | ---- | ----------- | ---------- |
 | {cpus} | Gauge | Double |
 
+### container.cpu.online
+
+Number of online CPUs.
```

Comment:
Is there any other documentation available from docker? It's hard to understand what this metric is needed for from this line.

Reply:
I can't find any documentation from Docker, only some references from the internet that says how people interpret this:
"The number of real used CPUs is given by cpu_stats.online_cpus (4 in our case)." - https://github.com/influxdata/telegraf/issues/2964
"The online_cpus value represents the number of CPU core on the host machine." - https://cratedb.com/blog/analyzing-docker-container-performance-native-tools
Even the source code doesn't say much: https://github.com/moby/moby/blob/460b4aebdf81512a3ea37c2a65826b890427537d/api/types/stats.go#L50

Reply:
Digging deeper into the code, this is the "cpu_num" return value here: https://github.com/moby/moby/blob/460b4aebdf81512a3ea37c2a65826b890427537d/daemon/stats_unix.go#L321
If we go to the `man` page in the comment, we can see that the command lists cpus with a number like `cpu0`, `cpu1` etc. and Docker just sums number of these different values.

Reply:
Ok, it seems like just a number of cores available to the container. We have `system.cpu.logical.count` for this purpose on the host, see https://github.com/open-telemetry/semantic-conventions/blob/6c79a1d9107c1833d95177beda84947ce1902792/model/metrics/system-metrics.yaml#L90C18-L90C42. Should we call it `container.cpu.logical.count`? I think _online_ here just adds confusion.  cc @open-telemetry/semconv-container-approvers

Reply:
It would make sense to introduce it as `container.cpu.logical_count` to be aligned with `system.*`, as per our discussions at https://github.com/open-telemetry/semantic-conventions/pull/282#issuecomment-1909845087.

Reply:
I have changed the name to `container.cpu.logical_count` as the suggestion said.

Reply:
Changed, thanks for responding quickly.

Reply:
@dmitryax could you rereview? 🙏